unime/src/routes/prompt/credential-offer/+page.svelte

```diff
@@ -69,11 +63,12 @@
     <div
       class="mt-3 w-full rounded-[20px] border border-slate-200 bg-white p-[10px] dark:border-slate-600 dark:bg-dark"
     >
-      {#each credential_offer.credentials as credential, index}
+      {#each Object.entries(credential_configurations) as [credential_configuration_id, credential_configuration]}
         <!-- TODO: careful with long list! -->
         <ListItemCard
-          id={`credential_${index}`}
-          title={$state?.current_user_prompt?.display?.at(index)?.name ?? credential.credential_definition.type.at(-1)}
+          id={`credential_${credential_configuration_id}`}
+          title={credential_configuration.display?.at(0).name ??
```

Comment:
When updating to `display: object[]`, we need to update `.at(0)` to `.at(0)?` to handle a possibly empty array.